<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="Rownanie_Friisa" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#Rownanie_Friisa{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(179, 204, 117, 255), stop:1 rgba(255, 255, 255, 255))};</string>
   </property>
   <widget class="QPushButton" name="oblicz_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>480</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Oblicz</string>
    </property>
   </widget>
   <widget class="QLabel" name="tekst_wyniki">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>540</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 15pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>WYNIK</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Wróć do głównego menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>480</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="tekstGlowny">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;Równanie Friisa&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="Dane_wejsciowe">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>230</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="gt_tekst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj zysk an. nadawczej (Gt):</string>
     </property>
    </widget>
    <widget class="QLabel" name="dBi_tekst">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[dBi]</string>
     </property>
    </widget>
    <widget class="QLabel" name="dBi_tekst_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[dBi]</string>
     </property>
    </widget>
    <widget class="QLabel" name="m_tekst">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[m]</string>
     </property>
    </widget>
    <widget class="QLabel" name="m_tekst_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[m]</string>
     </property>
    </widget>
    <widget class="QLabel" name="gr_tekst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj zysk an. odbiorczej (Gr):</string>
     </property>
    </widget>
    <widget class="QLabel" name="lambda_tekst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj długość fali cz. radiowej (λ):</string>
     </property>
    </widget>
    <widget class="QLabel" name="d_tekst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj odległość pomiędzy antenami (d):</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="gt_input">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>62</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999999.998999999952503</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="gr_input">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>62</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999999.998999999952503</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="lambda_input">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>62</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999999.998999999952503</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_input">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>62</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999999.998999999952503</double>
     </property>
    </widget>
    <widget class="QLabel" name="pr_tekst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj moc odbieraną (Pr):</string>
     </property>
    </widget>
    <widget class="QLabel" name="pt_tekst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Podaj moc nadajnika (Pt):</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pr_input">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>62</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999999.998999999952503</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pt_input">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>62</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>999999.998999999952503</double>
     </property>
    </widget>
    <widget class="QLabel" name="dBm_tekst">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[dBm]</string>
     </property>
    </widget>
    <widget class="QLabel" name="dBm_tekst_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[dBm]</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="tekst_podaj_dane">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>160</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 15pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>PODAJ DANE WEJŚCIOWE</string>
    </property>
   </widget>
   <widget class="QLabel" name="tekst_wybor_trybu">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 15pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>WYBIERZ TRYB</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Tryb_obliczen">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
gridline-color: rgb(255, 0, 4);</string>
    </property>
    <property name="title">
     <string>Tryb obliczeń</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="stosunek_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pr/Pt</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="pr_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pr</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="pt_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pt</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="rownanie_tekst">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>590</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wynik">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>580</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jednostka_tekst">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>590</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
